src/main/java/uk/gov/hmcts/reform/unspec/service/flowstate/StateFlowEngine.java

```diff
@@ -70,6 +70,7 @@ public StateFlow build() {
                 .transitionTo(CLAIM_DISCONTINUED).onlyIf(claimDiscontinued)
                 .transitionTo(CLAIM_WITHDRAWN).onlyIf(claimWithdrawn)
             .state(FULL_DEFENCE)
+                .transitionTo(CLAIM_STAYED)
```

Comment:
Do we want to have different internal states based on the path it took to reach CCD stayed state? 

- Scheduler moves the case to stayed
- claimant solicitor does not want to proceed, and we move this to stayed.

Reply:
That's a really good question @sabahirfan - I will review the flow states this week hopefully, also we have a story in the backlog to refactor all the states. We will pick this up again, for now whatever is implemented here is ok.